test/run-integration.js

```diff
@@ -164,6 +165,27 @@ function searchForFacetValues(t) {
     .then(noop, _.bind(t.error, t));
 }
 
+function searchForFacetValues(t) {
+  t.plan(2);
+
+  client
+    .searchForFacetValues([
+      {
```

Comment:
would maybe be interesting to test one with a single query only too, because I imagine that will be used more

Reply:
Thanks, I will add the test!